src/generator/operations.ts

```diff
@@ -681,7 +701,7 @@ function createProtocolResponse(op: Operation, imports: ImportManager): string {
       }
     }
     const mediaType = getMediaType(response.protocol);
-    if (mediaType === 'none' || mediaType === 'binary') {
+    if (mediaType === 'none' || mediaType === 'binary' || mediaType === 'text') {
```

Comment:
It might be better to flip the condition to marshal for `json` and `xml`.  That way we don't have to keep updating it if/when other media types are added (e.g. this will likely come up again when adding support for form data).

Reply:
done